tests/checkasm/vvc_sad.c

```diff
@@ -0,0 +1,191 @@
+#include <string.h>
```

Comment:
The license header is missing. Please refer to https://github.com/FFmpeg/FFmpeg/blob/bba996d6cdac15367f265e245987477d0f7b1899/tests/checkasm/vvc_mc.c#L2